packages/teams-js/src/internal/constants.ts

```diff
@@ -190,3 +190,98 @@ export const errorRuntimeNotSupported = 'The runtime version is not supported';
 
 /** @hidden */
 export const errorCallNotStarted = 'The call was not properly started';
+
+/**
+ * Use enum to set or update API version number
+ * Note: V_0 = 'v0' is used for APIs who needs to be passed with correct version number
+ * but haven't been implemented yet.
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export enum ApiVersionNumber {
+  V_0 = 'v0',
+  V_1 = 'v1',
+  V_2 = 'v2',
+  V_3 = 'v3',
+}
+
+export enum ApiName {
```

Comment:
This might be the same question, but are these meant to correspond to the teams-js name of the function or the function name that gets sent up to the host?

Reply:
It's to be corresponded to the teams-js name.

For example. we have dialog.url.open API in teams-js library but function name sent to hub sdks is tasks.startTask.

There might be easy to remember by one sentence: function names sent to Hub SDKs are aligned with API names in Teams-Js library V1 so for Teams-Js library V2, they are not necessarily required to be the same, and under most scenarios, they are different.

Reply:
So, if I may try restating what you wrote: the values in the enum ApiName correspond to the function names in teams-js, so, like Trevor said, if we update a function name in teams-js, we should update the ApiName as well, right? So if I have an api `myModule.doSomethingCool()`, then the ApiName should be "myModule.doSomethingCool". Then, if I rename it `myModule.doSomethingCooler()`, then the ApiName should be "myModule.doSomethingCooler", right? Then would the resulting version tags be "v1_myModule_doSomethingCool" and "v1_myModule_doSomethingCooler"?